File: front/src/interfaces/LogDto.ts
export interface LogDto {
  id?: string | null;
  refApplication?: string | null;
  category?: string | null;
  level?: string | null;
  message?: string | null;
  payloadJson?: string | null;
  fingerprint?: string | null;
  isPatched?: boolean | null;
  occurredAtUtc?: string | null;
  occurrences?: number | null;
}

File: api/DTOs/Log/LogDto.cs
namespace api.DTOs.Log
{
    public sealed class LogDto
    {
        public Guid? Id { get; set; }

        public Guid? RefApplication { get; set; }

        public string? Category { get; set; }

        public string? Level { get; set; }

        public string? Message { get; set; }

        public string? PayloadJson { get; set; }

        public string? Fingerprint { get; set; }

        public bool? IsPatched { get; set; }

        public DateTime? OccurredAtUtc { get; set; }

        public long? Occurrences { get; set; }
    }
}
